make/dev.mk

```diff
@@ -35,8 +35,7 @@ deploy-e2e-to-dev-namespaces-two-members:
 
 setup-dev-sso:
 	if [[ "${DEV_SSO}" == "true" ]]; then \
-		$(MAKE) download-assets ASSETS_FOLDER=scripts/ci/dev-sso && \
-		$(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; \
+		scripts/ci/setup-dev-sso.sh "--sso-ns $(DEV_SSO_NS)"; \
```

Comment:
I used `""` to group all the args but it is unnecessary. I clean up the `""` in 69ebb6562078f8b6fe8a036bbd3d482659917570. WDYT?

Reply:
looks better 👍  thanks. I proposed to switch also the parenthesis syntax (from make to bash)  since I saw you did it for the other scripts for example [here](https://github.com/codeready-toolchain/toolchain-e2e/pull/1063/files#diff-163703aeb25751c1605d305fbfdf39f3a9dfb74d2ac9d8a03f5cf0169d7d41b7R326).